astronomy/gravity_model.proto.txt

```diff
@@ -5,9 +5,7 @@ gravity_model {
   # of the IAU Working Group on Cartographic Coordinates and Rotational
```

Comment:
s/axis directions and mean radii/axis directions, mean radii, reference angles, and angular frequencies/